src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -152,7 +143,9 @@ export default Vue.extend<{}, {}, Computed, Props>({
                     const key = [current.areaId, ...this.nonAreaFilters.map(f => current.filterValues[f.id])].join("_");
                     if (!(key in displayRows)) {
                         const areaLabel =  this.flattenedAreas[current.areaId].label;
-                        const areaHierarchy = findPath(current.areaId, this.countryAreaFilterOption).replace(/undefined\//g, "")
+                        // The first replace removes the first layer of the path (ie, the country level)
```

Comment:
Would it make sense to start the recursion from `this.countryAreaFilterOption.children` in that case, so the country doesn't get included in the path in the first place? The country itself would have an undefined path, but that could easily be mapped to the empty string. 

Also seems like you could modify `findPath` so that the current area selected isn't included in the path, by returning an empty string when you find the matching area id: `if(id === obj[key]) return ""` (you'd just have to also change the `if (path)` check to `(if (path != undefined)` because the empty string is falsy.)